packages/rpc/src/methods/stacks/stx-contract-call.ts

```diff
@@ -0,0 +1,32 @@
+import { z } from 'zod';
+
+import { DefineRpcMethod, RpcRequest, RpcResponse } from '../../rpc/schemas';
+import { stacksTransactionDetailsSchema } from './_stacks-helpers';
+
+export const stxCallContractMethodName = 'stx_callContract';
```

Comment:
I think maybe this file should be rename to match the method? `stx-call-contract.ts`?